### ISSUE TITLE
Potential Maintenance hand-over

### ISSUE THREAD

**Issue by john-tadebois** [vanniktech/gradle-android-junit-jacoco-plugin#222]
Honestly, this is super unmaintaned. Happy for a person to jump in

_Originally posted by @vanniktech in https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/issues/217#issuecomment-1430425464_

**Comment by john-tadebois** [comment]
Hello @vanniktech , I would be interested in taking over the project. 

I don't know much about the internal workings of the plugin, but I use it in a project.

There are a few minor fixes to do before it can work with AGP 8+; a [pull request](https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/pull/219) already exists to that effect.

The main concern for this hand-over is the GitHub Actions to publish the plugin; will it work out of the box and publish to your plugin repository?

**Comment by vanniktech** [comment]
> The main concern for this hand-over is the GitHub Actions to publish the plugin; will it work out of the box and publish to your plugin repository?

Yes. If not I can look into it, but you should be able to just push a git tag and the release should kick off: https://github.com/vanniktech/gradle-android-junit-jacoco-plugin/tree/master/.github/workflows

**Comment by vanniktech** [comment]
I invited you

**Comment by vanniktech** [comment]
Closing in favor of #224
